tests/scripts/check-generated-files.sh

```diff
@@ -23,11 +23,29 @@
 
 set -eu
 
+if [ $# -ne 0 ] && [ "$1" = "--help" ]; then
+    cat <<EOF
+$0 [-u]
+This script checks that all generated file are up-to-date. If some aren't, by
+default the scripts reports it and exits in error; with the -u option, it just
+updates them instead.
+
+  -u    Update the files rather than return an error for out-of-date files.
+EOF
+    exit
+fi
+
 if [ -d library -a -d include -a -d tests ]; then :; else
     echo "Must be run from mbed TLS root" >&2
     exit 1
 fi
 
+UPDATE=
+if [ $# -ne 0 ] && [ "$1" = "-u" ]; then
+    shift
+    UPDATE='y'
+fi
```

Comment:
Might as well do this properly:
```
while getopts u OPTLET; do
    case $OPTLET in
        u) UPDATE=y;;
        \?) exit 1;;
    esac
done
```

Reply:
Since you approve the PR, I assume this was only an optional suggestion, so I'll skip it for now unless I have to rework the PR for another reason, but I'll keep that in mind in the future when adding options to shell scripts.